diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
index fdf8ef7268f298091ddb33566702bbc29425cf4d..7a8df98f0ce7ef5dbd2e2efe962cf6cce92174db 100644
--- a/ui/base/BUILD.gn
+++ b/ui/base/BUILD.gn
@@ -410,6 +410,7 @@ jumbo_component("base") {
     "//base:base_static",
     "//base:i18n",
     "//base/third_party/dynamic_annotations",
+    "//brave/ui/webui/resources",
     "//net",
     "//third_party/icu",
     "//third_party/zlib:zlib",
